qsimcirq_tests/qsimcirq_test.py

```diff
@@ -53,18 +53,38 @@ def test_cirq_qsim_simulate_fullstate(self):
         cirq.GridQubit(1, 0)
     ]
 
-    # Create a circuit
+    # Create a circuit.
     cirq_circuit = cirq.Circuit(
-        cirq.X(a)**0.5,  # Square root of X.
-        cirq.Y(b)**0.5,  # Square root of Y.
-        cirq.Z(c),  # Z.
-        cirq.CZ(a, d)  # ControlZ.
+        cirq.Moment([
+            cirq.X(a)**0.5,  # Square root of X.
+            cirq.H(b),       # Hadamard.
+            cirq.X(c),       # X.
```

Comment:
I think you should add a Y gate here. At present the resulting wavefunction has no complex entries, so you're not really testing that the float->complex conversion is working.

Reply:
Currently trying this with an S gate (to ensure that the complex component can't be extracted to the global phase) but I'm seeing issues. Might be related to #44.

Reply:
Done. Part of the issue was a global phase applied in qsim but not in Cirq; I've accounted for this in the test.